doc/src/user_guide.rst

```diff
@@ -1570,6 +1570,80 @@ Example::
     int-E = rho*(u*u + v*v + w*w)
     int-enst = rho*(%(vor1)s*%(vor1)s + %(vor2)s*%(vor2)s + %(vor3)s*%(vor3)s)
 
+.. _ascent-plugin:
+
+[soln-plugin-ascent]
+^^^^^^^^^^^^^^^^^^^^
+
+Uses `Alpine Ascent <https://github.com/Alpine-DAV/ascent>`_ to plot on-the-fly
+. You will need both Conduit and Ascent compiled with MPI and VTK-h support. The
```

Comment:
Do we need to repeat this part that is already in the installation guide?

Reply:
I'll move to the installation guide